content/english/python-tensorflow/activity-1.md

```diff
@@ -0,0 +1,83 @@
+---
+title: "Activity 1 - Importing the Tensorflow library and datasets"
+date: 2020-09-09T16:19:17-07:00
+weight: 2
+draft: false
+---
+
+
+<iframe width="100%" height="600px" src="https://www.youtube.com/embed/OxDn2xDXWi4" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>
+
+
+The first thing we have to do is to import the tensorflow library in order to use functions that will allow us to train your model.
```

Comment:
Please discuss further what these libraries are actually doing (perhaps in a tip box so that it's kinda like optional reading).

Reply:
*Add immediately after commented sentence* These libraries are essential as they are a collection of precompiled methods and functions that and importing them into our program allows us to access these methods without having to rewrite these entire libraries. For example, we import the methods and functions to TensorFlow and Numpy to avoid having to write the entirety of these program within our own program.